app/views/games/index.html.erb

```diff
@@ -27,7 +27,7 @@
         <h3>Join one of these games</h3>
       </div>
         <p> (list of all games. Have to add button to take user to that game still)
-        <h1><%= @list_all_games.inspect %></h1>
+        <h1><%= @games.inspect %></h1>
```

Comment:
Not the best way to do this.  Consider @games.each and do something with each game.  Review listing lessons in Flixter.